docs/06.docs/02.markdown/page.md

```diff
@@ -46,8 +46,8 @@ or:
 
 <pre>
 ```lucee+trycf
-&lt;cfset x = 10&gt;
```

Comment:
Can you double check this one? I think these ones are needed because it is showing how to use the documentation and wrapped in a `pre` tag. See here: https://docs.lucee.org/docs/markdown.html

Reply:
@DominicWatson you are correct! Fixing it